notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        Z = jax.random.normal(key,shape=(len(mu), n))
</pre><p>Same pep8 variable name for "Z"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>